lib/package.json

```diff
@@ -75,7 +75,8 @@
         "react": "^18.0.0",
         "react-dom": "^18.0.0",
         "react-router": "^6.2.0",
-        "react-router-dom": "^6.2.0"
+        "react-router-dom": "^6.2.0",
+        "typesense": "1.5.3"
```

Comment:
Could you move this dependency to peer dependencies?
So that only users who want to use it can pull it in.
Thanks!